numpy/core/tests/test_simd.py

```diff
@@ -156,6 +156,40 @@ def test_tobits(self):
             tobits = bin(self.tobits(vdata))
             assert tobits == bin(data_bits)
 
```

Comment:
Yay, tests!

Reply:
This one only tests the new universal intrinsics.